File: StudetDataBackup/5. Statistical Inference/t_test_1_samp.py
import pandas as pd
from scipy.stats import ttest_1samp

pg = pd.read_csv("PlantGrowth.csv")

ttest_1samp(pg['weight'], popmean=6, alternative="two-sided")

ttest_1samp(pg['weight'], popmean=6, alternative="greater")

ttest_1samp(pg['weight'], popmean=6, alternative="less")

############## CO2 ##################

co2 = pd.read_csv("CO2.csv")

ttest_1samp(co2['uptake'], popmean=30, alternative="less")

# Conclusion: Pop mean uptake may be less than 30.

######### Indometh ################
indometh = pd.read_csv("Indometh.csv")
ttest_1samp(indometh['conc'], popmean=0.3, alternative="greater")
# Conclusion: Plasma Conc may be greater than 0.3.

File: StudetDataBackup/5. Statistical Inference/t_test_1_samp.R
setwd("C:/Training/Academy/Statistics (Python)/Datasets")
pg = read.csv("PlantGrowth.csv")

t.test(pg$weight, mu=6, alternative = 't')
t.test(pg$weight, mu=6, alternative = 'g')
t.test(pg$weight, mu=6, alternative = 'l')


co2 <- read.csv("CO2.csv")
t.test(co2$uptake, mu=30, alternative = 'l')

data(Indometh)
t.test(Indometh$conc, mu=0.3, alternative = "g")
